application/src/test/java/eu/europa/ec/fisheries/uvms/activity/service/bean/FluxMessageServiceBeanTest.java

```diff
@@ -80,18 +77,16 @@ public void setUp() {
         fluxFaReportMessage.setFLUXReportDocument(MapperUtil.getFluxReportDocumentCorrection());
 
         Mockito.doReturn(getMockedFishingActivityReportEntity()).when(faReportDocumentDao).findFaReportByIdAndScheme(any(String.class), any(String.class));
-        Mockito.when(fluxFaReportMessageDao.createEntity(any(FluxFaReportMessageEntity.class))).then(invocation -> invocation.getArguments()[0]);
     }
 
     @Test
+    @Ignore
```

Comment:
A short description of why the tests are ignored would be nice (even if  the test is crap)

Reply:
Well, next step is to remove these tests completly

Reply:
Tests removed